README.md

```diff
@@ -19,3 +19,5 @@ The package can be installed via `pip`:
 ```
 pip install [--user] tuneit
 ```
+
```

Comment:
Here would be good to have:
- a section documentation, with a link to the webpage and an extract of the first page
- a very simple example without too much text (sparse is fine, just put the code together and show only benchmark)
- and the following acknowledgements
```
## Acknowledgments

### Authors
- Simone Bacchio (sbacchio)
- Raphaella Demetriou (raphdem)

### Fundings
- PRACE-6IP, Grant agreement ID: 823767, Project name: LyNcs.
```